addon/configuration.ts

```diff
@@ -0,0 +1,29 @@
+// @ts-nocheck
```

Comment:
Keep `@ts-nocheck` here?

Reply:
I copied the file from publishr-admin I think, If I remove it I get a lot of warnings 🤷